po/es.po

```diff
@@ -664,32 +674,32 @@ msgid ""
 "Try ``%s --help'' for more information.\n"
 msgstr ""
 "Argumentos no reconocidos.\n"
-"Intente ``%s --help'' para más información.\n"
+"Pruebe ``%s --help'' para obtener más información.\n"
 
 #: hardinfo/util.c:903
 #, c-format
 msgid "Module \"%s\" depends on module \"%s\", load it?"
-msgstr "El módulo \"%s\" depende del módulo \"%s\", ¿desea cargarlo?"
+msgstr "El módulo «%s» depende del módulo «%s»; ¿desea cargarlo?"
```

Comment:
`\"%s\"` changed to `«%s» ` this actually works? please check after compile and run...

Reply:
Why shouldn't it work? They're normal Spanish quotation marks. I test as much as I can but it's not always possible to test everything because some strings only show after very specific actions or errors.
https://en.wikipedia.org/wiki/Quotation_mark#Spanish